2024-05-21 07:51:43,935 INFO    StreamThr :169650 [internal.py:wandb_internal():86] W&B internal server running at pid: 169650, started at: 2024-05-21 07:51:43.935263
2024-05-21 07:51:43,937 DEBUG   HandlerThread:169650 [handler.py:handle_request():146] handle_request: status
2024-05-21 07:51:43,942 INFO    WriterThread:169650 [datastore.py:open_for_write():87] open: /home/uosai/Desktop/on-policy/onpolicy/scripts/results/StarCraft2/3m/rmappo/Original_Mappo/wandb/run-20240521_075143-bdzp7azy/run-bdzp7azy.wandb
2024-05-21 07:51:43,943 DEBUG   SenderThread:169650 [sender.py:send():382] send: header
2024-05-21 07:51:43,964 DEBUG   SenderThread:169650 [sender.py:send():382] send: run
2024-05-21 07:51:44,481 INFO    SenderThread:169650 [dir_watcher.py:__init__():211] watching files in: /home/uosai/Desktop/on-policy/onpolicy/scripts/results/StarCraft2/3m/rmappo/Original_Mappo/wandb/run-20240521_075143-bdzp7azy/files
2024-05-21 07:51:44,481 INFO    SenderThread:169650 [sender.py:_start_run_threads():1136] run started: bdzp7azy with start time 1716245503.936296
2024-05-21 07:51:44,493 DEBUG   HandlerThread:169650 [handler.py:handle_request():146] handle_request: check_version
2024-05-21 07:51:44,493 DEBUG   SenderThread:169650 [sender.py:send_request():409] send_request: check_version
2024-05-21 07:51:44,564 DEBUG   HandlerThread:169650 [handler.py:handle_request():146] handle_request: run_start
2024-05-21 07:51:44,574 DEBUG   HandlerThread:169650 [system_info.py:__init__():27] System info init
2024-05-21 07:51:44,574 DEBUG   HandlerThread:169650 [system_info.py:__init__():42] System info init done
2024-05-21 07:51:44,574 INFO    HandlerThread:169650 [system_monitor.py:start():194] Starting system monitor
2024-05-21 07:51:44,574 INFO    SystemMonitor:169650 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-05-21 07:51:44,574 INFO    HandlerThread:169650 [system_monitor.py:probe():214] Collecting system info
2024-05-21 07:51:44,575 INFO    SystemMonitor:169650 [interfaces.py:start():190] Started cpu monitoring
2024-05-21 07:51:44,575 INFO    SystemMonitor:169650 [interfaces.py:start():190] Started disk monitoring
2024-05-21 07:51:44,576 INFO    SystemMonitor:169650 [interfaces.py:start():190] Started gpu monitoring
2024-05-21 07:51:44,577 INFO    SystemMonitor:169650 [interfaces.py:start():190] Started memory monitoring
2024-05-21 07:51:44,577 INFO    SystemMonitor:169650 [interfaces.py:start():190] Started network monitoring
2024-05-21 07:51:44,589 DEBUG   HandlerThread:169650 [system_info.py:probe():151] Probing system
2024-05-21 07:51:44,596 DEBUG   HandlerThread:169650 [system_info.py:_probe_git():136] Probing git
2024-05-21 07:51:44,614 DEBUG   HandlerThread:169650 [system_info.py:_probe_git():144] Probing git done
2024-05-21 07:51:44,614 DEBUG   HandlerThread:169650 [system_info.py:probe():199] Probing system done
2024-05-21 07:51:44,614 DEBUG   HandlerThread:169650 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-35-generic-x86_64-with-glibc2.35', 'python': '3.9.18', 'heartbeatAt': '2024-05-20T22:51:44.589730', 'startedAt': '2024-05-20T22:51:43.930279', 'docker': None, 'cuda': None, 'args': ('--seed', '305', '--env_name', 'StarCraft2', '--experiment_name', 'Original_Mappo', '--group_name', 'Original_Mappo', '--use_reward_shaping', '--use_shaping_weight', '--num_gpu', '0'), 'state': 'running', 'program': '/home/uosai/Desktop/on-policy/onpolicy/scripts/train/train_smac.py', 'codePathLocal': 'train_smac.py', 'codePath': 'onpolicy/scripts/train/train_smac.py', 'git': {'remote': 'https://github.com/narame7/on-policy.git', 'commit': 'a843a882a6e81ddd9b6e30e03a1d91e4076549c9'}, 'email': 'singfor7012@gmail.com', 'root': '/home/uosai/Desktop/on-policy', 'host': 'uos-desktop', 'username': 'uosai', 'executable': '/home/uosai/Desktop/miniconda3/envs/marl/bin/python', 'cpu_count': 12, 'cpu_count_logical': 24, 'cpu_freq': {'current': 4189.7849166666665, 'min': 2200.0, 'max': 3700.0}, 'cpu_freq_per_core': [{'current': 4157.598, 'min': 2200.0, 'max': 3700.0}, {'current': 3669.19, 'min': 2200.0, 'max': 3700.0}, {'current': 4561.756, 'min': 2200.0, 'max': 3700.0}, {'current': 4586.281, 'min': 2200.0, 'max': 3700.0}, {'current': 4190.26, 'min': 2200.0, 'max': 3700.0}, {'current': 4566.469, 'min': 2200.0, 'max': 3700.0}, {'current': 4516.296, 'min': 2200.0, 'max': 3700.0}, {'current': 4585.639, 'min': 2200.0, 'max': 3700.0}, {'current': 4223.557, 'min': 2200.0, 'max': 3700.0}, {'current': 4555.607, 'min': 2200.0, 'max': 3700.0}, {'current': 4008.612, 'min': 2200.0, 'max': 3700.0}, {'current': 3724.772, 'min': 2200.0, 'max': 3700.0}, {'current': 4585.08, 'min': 2200.0, 'max': 3700.0}, {'current': 3649.765, 'min': 2200.0, 'max': 3700.0}, {'current': 3751.169, 'min': 2200.0, 'max': 3700.0}, {'current': 4567.132, 'min': 2200.0, 'max': 3700.0}, {'current': 4321.094, 'min': 2200.0, 'max': 3700.0}, {'current': 4149.879, 'min': 2200.0, 'max': 3700.0}, {'current': 3942.906, 'min': 2200.0, 'max': 3700.0}, {'current': 4577.819, 'min': 2200.0, 'max': 3700.0}, {'current': 3947.724, 'min': 2200.0, 'max': 3700.0}, {'current': 4156.859, 'min': 2200.0, 'max': 3700.0}, {'current': 4139.576, 'min': 2200.0, 'max': 3700.0}, {'current': 4552.799, 'min': 2200.0, 'max': 3700.0}], 'disk': {'/': {'total': 233.17998886108398, 'used': 178.75580215454102}}, 'gpu': 'NVIDIA GeForce RTX 3080', 'gpu_count': 2, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3080', 'memory_total': 10737418240}, {'name': 'NVIDIA GeForce RTX 3080', 'memory_total': 10737418240}], 'memory': {'total': 62.70906448364258}}
2024-05-21 07:51:44,614 INFO    HandlerThread:169650 [system_monitor.py:probe():224] Finished collecting system info
2024-05-21 07:51:44,614 INFO    HandlerThread:169650 [system_monitor.py:probe():227] Publishing system info
2024-05-21 07:51:44,614 DEBUG   HandlerThread:169650 [system_info.py:_save_conda():208] Saving list of conda packages installed into the current environment
2024-05-21 07:51:45,483 INFO    Thread-12 :169650 [dir_watcher.py:_on_file_created():271] file/dir created: /home/uosai/Desktop/on-policy/onpolicy/scripts/results/StarCraft2/3m/rmappo/Original_Mappo/wandb/run-20240521_075143-bdzp7azy/files/conda-environment.yaml
2024-05-21 07:51:46,248 DEBUG   HandlerThread:169650 [system_info.py:_save_conda():220] Saving conda packages done
2024-05-21 07:51:46,249 INFO    HandlerThread:169650 [system_monitor.py:probe():229] Finished publishing system info
2024-05-21 07:51:46,252 DEBUG   SenderThread:169650 [sender.py:send():382] send: files
2024-05-21 07:51:46,252 INFO    SenderThread:169650 [sender.py:_save_file():1403] saving file wandb-metadata.json with policy now
2024-05-21 07:51:46,257 DEBUG   HandlerThread:169650 [handler.py:handle_request():146] handle_request: python_packages
2024-05-21 07:51:46,257 DEBUG   SenderThread:169650 [sender.py:send_request():409] send_request: python_packages
2024-05-21 07:51:46,257 DEBUG   HandlerThread:169650 [handler.py:handle_request():146] handle_request: stop_status
2024-05-21 07:51:46,258 DEBUG   SenderThread:169650 [sender.py:send_request():409] send_request: stop_status
2024-05-21 07:51:46,259 DEBUG   HandlerThread:169650 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-21 07:51:46,481 INFO    Thread-12 :169650 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/uosai/Desktop/on-policy/onpolicy/scripts/results/StarCraft2/3m/rmappo/Original_Mappo/wandb/run-20240521_075143-bdzp7azy/files/conda-environment.yaml
2024-05-21 07:51:46,482 INFO    Thread-12 :169650 [dir_watcher.py:_on_file_created():271] file/dir created: /home/uosai/Desktop/on-policy/onpolicy/scripts/results/StarCraft2/3m/rmappo/Original_Mappo/wandb/run-20240521_075143-bdzp7azy/files/wandb-metadata.json
2024-05-21 07:51:46,482 INFO    Thread-12 :169650 [dir_watcher.py:_on_file_created():271] file/dir created: /home/uosai/Desktop/on-policy/onpolicy/scripts/results/StarCraft2/3m/rmappo/Original_Mappo/wandb/run-20240521_075143-bdzp7azy/files/requirements.txt
2024-05-21 07:51:46,590 DEBUG   SenderThread:169650 [sender.py:send():382] send: telemetry
2024-05-21 07:51:46,639 ERROR   wandb-upload_0:169650 [internal_api.py:execute():373] 404 response executing GraphQL.
2024-05-21 07:51:46,639 ERROR   wandb-upload_0:169650 [internal_api.py:execute():374] {"errors":[{"message":"run StarCraft2/bdzp7azy not found during createRunFiles","path":["createRunFiles"]}],"data":{"createRunFiles":null}}
2024-05-21 07:51:46,639 ERROR   Thread-11 :169650 [step_upload.py:_handle_event():200] Failed to upload file: /tmp/tmp2nyq2_srwandb/ghno3y2d-wandb-metadata.json
Traceback (most recent call last):
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 106, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2384, in upload_urls
    query_result = self.gql(
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 341, in gql
    ret = self._retry_gql(
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/uosai/Desktop/miniconda3/envs/marl/lib/python3.9/site-packages/wandb/util.py", line 907, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2024-05-21 07:51:49,591 DEBUG   HandlerThread:169650 [handler.py:handle_request():146] handle_request: status_report
2024-05-21 07:51:50,565 INFO    MainThread:169650 [internal.py:handle_exit():76] Internal process exited
2024-05-21 07:51:50,580 INFO    memory    :169650 [interfaces.py:monitor():140] Process proc.memory.rssMB has exited.
2024-05-21 07:51:50,580 INFO    cpu       :169650 [interfaces.py:monitor():140] Process cpu has exited.
2024-05-21 07:51:50,580 DEBUG   SystemMonitor:169650 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-05-21 07:51:50,580 DEBUG   SystemMonitor:169650 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-05-21 07:51:50,580 DEBUG   SystemMonitor:169650 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-05-21 07:51:50,581 DEBUG   SenderThread:169650 [sender.py:send():382] send: stats
